README.md

```diff
@@ -69,7 +69,7 @@ To create your first LoopBack 4 application, see
 ## Documentation
 
 - [Official documentation](http://loopback.io/doc/en/lb4/)
-- [API documentation](http://apidocs.loopback.io/#LoopBack4)
+- [API documentation](https://loopback.io/doc/en/lb4/apidocs.index.html)
 - [FAQ](http://loopback.io/doc/en/lb4/FAQ.html)
 - [LoopBack 3 vs LoopBack 4](http://loopback.io/doc/en/lb4/LoopBack-3.x.html)
```

Comment:
While you're here, I think this page still exists but not in the sidebar. Perhaps we can change it to the migration guide? https://loopback.io/doc/en/lb4/migration-overview.html. thanks.

Reply:
Oh I actually just created a PR https://github.com/strongloop/loopback-next/pull/5013 for this but I linked to https://loopback.io/doc/en/lb4/Understanding-the-differences.html instead. Do you think https://loopback.io/doc/en/lb4/migration-overview.html would be better?

Reply:
ah. didn't see your other PR. :) 
Since the migration guide is at the top level, I think it might be better to put the migration guide instead.